src/app/components/App.tsx

```diff
@@ -447,6 +448,7 @@ const App = () => {
       <ErrorBoundary fallback={null}>
         <GDPR />
       </ErrorBoundary>
+      <ContextualHelpHandler />
```

Comment:
I'd probably wrap that in a similar error boundary as above, we don't want anything problematic in there crashing the whole page

Reply:
I would also probably put in a standalone `Route` matching not /help and not /contact (and anything else that you think about will not need contextual help in order to not display the button and not load the corresponding logic